packages/experimental/Drawer/src/Drawer.types.ts

```diff
@@ -0,0 +1,195 @@
+import type { Animated, ColorValue, FlexAlignType, ModalProps, TouchableWithoutFeedbackProps, ViewProps } from 'react-native';
+
+import type { InteractionEvent, PressableFocusProps } from '@fluentui-react-native/interactive-hooks';
+
+export const DrawerName = 'Drawer';
+
+export type DrawerPositionType = 'left' | 'right' | 'bottom';
+
+export interface DrawerTokens {
+  /**
+   * The color of the backdrop.
+   * */
+  backdropColor?: ColorValue;
+
+  /**
+   * Width of the handle.
+   */
+  handleWidth?: number;
+
+  /**
+   * Height of the handle.
+   */
+  handleHeight?: number;
+
+  /**
+   * The background color of the handle.
+   * */
+  handleBackgroundColor?: ColorValue;
+
+  /**
+   * The corner radius of the handle.
+   * */
+  handleCornerRadius?: number;
+
+  /**
+   * The top margin of the handle.
+   *  */
+  handleMarginTop?: number;
+
+  /**
+   * The bottom margin of the handle.
+   * */
+
+  handleMarginBottom?: number;
+
+  /**
+   * The start margin of the handle.
+   * */
+
+  handleMarginStart?: number;
+
+  /**
+   * The end margin of the handle.
+   * */
+
+  handleMarginEnd?: number;
+
+  /**
+   * The flex alignment of the handle.
+   * */
+  handleAlignment?: FlexAlignType;
+
+  /**
+   * The background for drawer
+   * */
+
+  drawerBackgroundColor?: ColorValue;
+
+  /**
+   * The corner radius of the Drawer
+   * @default 0
+   * */
+  drawerCornerRadius?: number;
+
+  /**
+   * The elevation of the Drawer
+   * @default 5
+   * */
+  drawerElevation?: number;
+
+  /**
+   * The position of the Drawer
+   * @default 'left'
+   * */
+  position?: 'left' | 'right' | 'bottom';
+
+  /**
+   * The width of the Drawer
+   * Note: Only applicable when position is 'left' or 'right'
+   * @default '100%'
+   * */
+  width?: number | string;
+
+  /**
+   * The height of the Drawer
+   * Note: Only applicable when position is 'bottom'
+   * @default '40%'
+   * */
+  height?: number | string;
+
+  /**
+   * The shadow color of the Drawer
+   * @platform iOS
+   * */
+  shadowColor?: ColorValue;
+
+  /**
+   * The shadow offset of the Drawer
+   * @platform iOS
+   * */
+  shadowOffset?: { width: number; height: number };
+
+  /**
+   * The shadow opacity of the Drawer
+   * @platform iOS
+   * */
+  shadowOpacity?: number;
+
+  /**
+   * The shadow radius of the Drawer
```

Comment:
cleanups pre merge

Reply:
hmm , didn't get it, do you mean remove this ?

Reply:
the @platform iOS *

Reply:
But it is the property required for iOS for the shadow, i.e - elevation on android.  
I have them so that they can reused when we have a iOS component, For the spec i think the guidance is to consider other platform as well.

Reply:
The guidance has been to test basic functionality for any new pure JS control in iOS, so that it validates bare-bones code sharing. If this token is not used currently - IMO it's something to skip for when the iOS fit and finish comes. Keeping types available which are non functional/ do not have any impact might add usage confusion.

Reply:
Taking it to next PR  to make it sanitized to Android.